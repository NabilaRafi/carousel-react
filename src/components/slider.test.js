import React from 'react';
import { create } from 'react-test-renderer';
import Slider from './slider';

const suite = 'Testing slider component <SlideItem />';

describe(suite, () => {
    const component = create.apply(<Slider />).toJSON();
    test('renders <slideItem /> component without crashing', () => {
        expect(component).toMatchSnapshot();
    });
})