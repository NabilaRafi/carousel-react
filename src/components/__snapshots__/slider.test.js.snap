// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing slider component <SlideItem /> renders <slideItem /> component without crashing 1`] = `null`;
